.csscomb.json

```diff
@@ -0,0 +1,32 @@
+{
+  "always-semicolon": true,
+  "block-indent": "    ",
+  "color-case": "upper",
+  "color-shorthand": true,
+  "element-case": "lower",
+  "eof-newline": true,
+  "leading-zero": true,
+  "quotes": "double",
+  "remove-empty-rulesets": true,
+  "space-after-combinator": " ",
+  "space-after-opening-brace": "\n",
+  "space-after-selector-delimiter": " ",
```

Comment:
Is this what is causing the closing brace to be on the same line as the last property ?